installer/parse_package_xml.py

```diff
@@ -0,0 +1,74 @@
+#! /usr/bin/env python3
+
+import os
+import sys
+import xml.etree.ElementTree as ET
+
+from catkin_pkg.condition import evaluate_condition
+
+
+def main():
+    if len(sys.argv) != 2:
+        print("Usage: parse_package_xml PACKAGE.XML")
+        return 1
+
+    print("\n".join(packagexml_parser(sys.argv[1])["deps"]))
```

Comment:
```suggestion
    print("\n".join(packagexml_parser(sys.argv[1])["deps"]))
    return 0
```

Reply:
This should not be required

Reply:
No, not required, just consistent return value.